disable button on hover

fn-faisal opened the pull request.

davimacedo commented:
Not sure. I unserstand it will remove the text-decoration always. We wanted to remove it only when the button is disabled, right?

fn-faisal commented:
@davimacedo  I added it because I thought the task was to removed the text decoration all together. I've updated it to remove text-decoration only from disabled state.
